### ISSUE TITLE
startmode on stop

### ISSUE THREAD

**Issue by arnulfholm** [kohsuke/winsw#251]
In the xml file I can set the start mode of the service with <startmode>demand</startmode>.

But what I would like to be able to do is this:
* when I start the service, startmode should be auto
* when I stop the service, startmode should be demand (to make sure that service is not started again if the windows server restarts)

Is this possible now?

**Comment by oleg-nenashev** [comment]
No, it's not possible in WinSW, but technically it could be implemented in a compatible way (by adding opt-in flag). Feel free to propose a pull request

**Comment by arnulfholm** [comment]
Ok, but I worked around it, and I feel the solution is better than to change the startmode on stop.
What I do now is:
1. Install the service
2. Start the service

And when I'm going to shutdown the service:
1. Stop the service
2. Uninstall the service

**Comment by sladyn98** [comment]
@oleg-nenashev As discussed in the gsoc-sig meeting I would like to start with some beginner friendly issues.
In the defaultSettings.cs the default for the startMode is already auto, when the service stops can we change the mode to demand in `StopIt()`.
```
                  public StartMode StartMode { get { return StartMode.Automatic; } }
```

**Comment by NextTurn** [comment]
I don't think this feature is related to the existing `stop` command though. Maybe new commands like `enable`/`disable` or `set` which allows more modifications on demand.
